rosettanet/2.0/modules/ROOT/pages/rosettanet-connector-examples.adoc

```diff
@@ -467,7 +468,7 @@ You can download a https://s3-us-west-2.amazonaws.com/mulesoft-sites-vendorconte
 
 For example, the following `curl` command posts a purchase order confirmation:
 
-`curl -v -H "Content-Type: application/text" -XPOST --data-binary @sample-purchase-order-confirmation-content.xml http://localhost:8802`
+`curl -v -H "Content-Type: application/text" -XPOST --data-binary @sample-purchase-order-confirmation-content.xml http://localhost:8082`
```

Comment:
This should be port `8802` as this curl is to trigger the send operation flow, I would also change the `application/text` to `application/xml`
```suggestion
`curl -v -H "Content-Type: application/xml" -XPOST --data-binary @sample-purchase-order-confirmation-content.xml http://localhost:8802`
```